contracts/tests/sbtc-token.test.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  alice,
+  bob,
+  deposit,
+  token,
+} from "./helpers";
+import { test, expect, describe } from "vitest";
+import { txOk, filterEvents, rov } from "@clarigen/test";
+import { CoreNodeEventType, cvToValue } from '@clarigen/core';
+
+describe("sBTC token contract", () => {
```

Comment:
I don't think we need to get full code coverage in this PR, but I'd like to see some tests around "only protocol caller" functions. Basically just call them with Alice as sender and verify the error code

Reply:
Alternatively, I'm ok with us making a ticket to add unit tests for proper coverage of the sBTC token contract

Reply:
Added two more tests to hit coverage on the two existing error constants.